src/components/Keyword/AddKeyword/index.js

```diff
@@ -0,0 +1,188 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import KeywordHeader from '../KeywordHeader';
+import { getSiteRecommendation, getKeywordRecommendation } from 'store/modifyKeyword';
+import { inquiry } from 'store/keyword';
+import * as S from './styles';
+import { useDispatch, useSelector } from 'react-redux';
+import KeywordAlarm from '../KeywordAlarm';
+
+const AddKeyword = () => {
+  const [site, setSite] = useState('');
+  const [recommendKeyword, setRecommendKeyword] = useState('');
+  const [recommendList, setRecommendList] = useState([]);
+  const [recommendKeywords, setRecommendKeywords] = useState([]);
+  const [selectRecommendItem, setSelectRecommendItem] = useState([]);
+  const [selectRecommendKeyword, setSelectRecommendKeyword] = useState('');
+  const [alreadyRegisterItem, setAlreadyRegisterItem] = useState(false);
+  const [alreadyRegisterKeyword, setAlreadyRegisterKeyword] = useState(false);
```

Comment:
```suggestion
  const [isRegisterItem, setIsRegisterItem] = useState(false);
  const [isRegisterKeyword, setIsRegisterKeyword] = useState(false);
```
boolean 값을 다루니 이런식으로 작성하는게 더 좋을 것 같습니다.

Reply:
앞으로 Boolean 값 작성시 is를 사용하도록 하겠습니다

Reply:
[8daeb6c](https://github.com/BCSDLab/KOALA_FRONT_END/pull/47/commits/8daeb6c17c581cac1832366d507f754f89b3cf66)